src/test/java/task40/TableSortSearchTest.java

```diff
@@ -1,47 +1,53 @@
 package task40;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.support.ui.Select;
+import task40.helpes.BaseTest;
 import task40.helpes.Employee;
 import task40.helpes.TableSortSearchPage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class TableSortSearchTest {
-    private WebDriver driver;
+import static task40.helpes.TestConstants.*;
+
+public class TableSortSearchTest extends BaseTest {
     private final int ageOption = 25;
     private final int salaryOption = 150000;
+    private final String tableCssSelector = "table#example tbody";
```

Comment:
Test class shouldn't know anything about xpath'es/locatoros. It should be a part of page class

Reply:
Done